VimeoNetworking/Sources/ExceptionCatcher+Swift.swift

```diff
@@ -35,7 +35,7 @@ class ExceptionCatcher: ObjC_ExceptionCatcher
      
      - throws: an error containing any thrown exception information
      */
-    @nonobjc internal static func doUnsafe(unsafeBlock: @escaping ((Void) -> Void)) throws
+    @nonobjc public static func doUnsafe(unsafeBlock: @escaping (() -> Void)) throws
```

Comment:
Why public for this class and method?

Reply:
**ObjC_ExceptionCatcher** is Objective-C, of course. When we subclass in Swift we end up with the default access of`internal`, which makes sense. And as you pointed out below this should be elevatated to `open` when using `@testable` but… it's not. 

I've tried working around this but the best I could come up with was to just mark it as `public`. Maybe I'm missing something? Let me know if you want chat about it.